change-notes/1.23/analysis-python.md

```diff
@@ -3,7 +3,19 @@
 
 ## General improvements
 
+### Python 3.8 support
 
+Python 3.8 syntax is now supported. In particular, the following constructs are parsed correctly:
+
+- Assignment expressions using the "walrus" operator, such as `while chunk := file.read(1024): ...`.
+- The positional argument separator `/`, such as in `def foo(a, /, b, *, c): ...`.
+- Self-documenting expressions in f-strings, such as `f"{var=}"`.
+
+### General query improvements
+
+In an effort to deprecate the `Object` API (e.g. `ClassObject`) in favour of the
```

Comment:
My suggestion here and in the line below is intended to state this more positively.
```suggestion
Following the replacement of the `Object` API (for example, `ClassObject`) in favor of the
```